### PR TITLE
first draft of Q&A page

### PR DESCRIPTION
**Pull request by schottj**
## Description & motivation
This guide should help users upgrade dbt versions in cloud and highlights breaking changes.

## Checklist
If you added new pages (delete if not applicable):
- [X] The page has been added to `website/sidebars.js`
- [X] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (jtcohen6) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: Might be worth adding a note here to the effect of, we know this information isn't exposed as well it could be, we're thinking of ways to make it better
```suggestion
If you use any packages from [dbt Hub]([https://hub.getdbt.com/](https://hub.getdbt.com/)), make sure you also upgrade to a version of the package that supports the dbt version you intend to upgrade to. You can see which dbt versions a package supports by checking on the `require-dbt-version:` in the package's dbt_project.yml file on Github.

As an example, dbt-utils version 0.7.1 supports dbt v0.20, and it will support v0.21 when it is released, as described in its [dbt_project.yml](https://github.com/dbt-labs/dbt-utils/blob/0.7.1/dbt_project.yml).

After you've changed the package version in your packages.yml file, be sure to run `dbt deps` in the IDE to install the updated version.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (jtcohen6) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: 👍 

Maybe: `See "Upgrading to v0.17 from v0.16" below for more details.`

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (jtcohen6) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: Do you want to discuss creating a new project with a brand-new development environment, to avoid causing issues for other developers using the IDE at the same time?
```suggestion
Once you have an idea about what code changes you'll need to make, you can start implementing them in your development environment. You'll first need to [change the project's development environment settings](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version#supported-versions) to run the latest version of dbt. Next check out a branch `dbt-version-upgrade`, make the appropriate updates to your project (if needed), and see if your dbt project compiles and runs with the new version of dbt in the IDE. If jumping directly to the latest version of dbt is too far of a leap for your project, try iteratively getting your project to work on each successive minor version. There are years of development and a handful of breaking changes between two distant versions of dbt (e.g. 0.14 --> 0.20). There are far fewer between two subsequent versions of dbt, which is why upgrading regularly is important.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (jtcohen6) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: With an eye toward building the habit of thinking in minor versions (and _always_ using latest patch releases), what do you think of changing this one (and following):

- Instead of: `Upgrading to v0.20.0 from v0.19.x`
- We just say: `Upgrading to v0.20 from v0.19`

Where those imply `v0.20.latest` and `v0.19.latest`

I know the migration guide is called "Upgrading to 0.20.0"; I'm open to changing them to "Upgrading to v0.20" instead.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (jtcohen6) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]:
```suggestion
title: "Upgrading dbt versions in Cloud"
```
I think?

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (jtcohen6) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: This feels like more of an `:::info` (read this!) than a `:::caution` (bad things might/will happen!), but I don't know our latest thinking on callout style

**Review by jtcohen6 — COMMENTED** [review]
@schottj This is awesome! Thanks so much for pulling it together. I left some comments/suggestions, some of which are pretty nit-picky, none of which should block merging this.

I'm also going to make a mental note for myself to add to this page in my next prerelease docs update (#767).

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (schottj) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: I might actually introduce the .latest suffix.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (schottj) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: I felt this was more important than just info, but I could also change it

Comment (jtcohen6) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md]: yeah I was going back and forth. happy either way!

**Comment by schottj** [comment]
@barberscott Would appreciate your thoughts when you get some time!

**Comment by schottj** [comment]
@barberscott @alexbethea22 @jtcohen6 @drewbanin could I get a second pair of eyes and an approval please?

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md
Comment (alexbethea22) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md:32]: I think these are formatting weird in the preview! 
image

'Upgrading to v0.19.latest from v0.18' and 'Upgrading to v0.17.latest from v0.16' both look good in the preview, but all others don't seem to be formatted correctly!

**Review by alexbethea22 — COMMENTED** [review]
Left one comment! 

This looks great! Thanks for putting this together 🙌

**Comment by schottj** [comment]
@alexbethea22 fixed! Thank you! If it looks good can I get an approval?

**Review by alexbethea22 — APPROVED** [review]
LGTM!

**Pull request merged**
